release/release_versions.sh

```diff
@@ -1,9 +1,9 @@
 #!/usr/bin/env bash
 
 # Make sure to update these versions when doing a release first
-PYTORCH_VERSION=${PYTORCH_VERSION:-2.1.1}
-TORCHVISION_VERSION=${TORCHVISION_VERSION:-0.16.1}
-TORCHAUDIO_VERSION=${TORCHAUDIO_VERSION:-2.1.1}
-TORCHTEXT_VERSION=${TORCHTEXT_VERSION:-0.16.1}
+PYTORCH_VERSION=${PYTORCH_VERSION:-2.2.0}
+TORCHVISION_VERSION=${TORCHVISION_VERSION:-0.17.0}
+TORCHAUDIO_VERSION=${TORCHAUDIO_VERSION:-2.2.0}
+TORCHTEXT_VERSION=${TORCHTEXT_VERSION:-0.17.0}
 TORCHDATA_VERSION=${TORCHDATA_VERSION:-0.7.1}
-TORCHREC_VERSION=${TORCHREC_VERSION:-0.6.0}
\ No newline at end of file
+TORCHREC_VERSION=${TORCHREC_VERSION:-0.6.0}
```

Comment:
Give me a sec to land the fbgemm PR https://github.com/pytorch/builder/pull/1681, may be this will complain about a conflict and ask for a rebase.